app/src/main/java/com/netmera/netmerafintech/ui/home/ManageCardActivity.kt

```diff
@@ -59,9 +64,16 @@ class ManageCardActivity: AppCompatActivity() {
         }
     }
 
+    private fun onBackAction() {
+        intent?.data?.let {
+            startActivity(Intent(this@ManageCardActivity, AllPagesActivity::class.java))
+        }
+        finish()
```

Comment:
I think this should be like:
```suggestion
        super.onBackPressed()
```

Reply:
Backstack is empty when we open the page through deep link, so we override onBackPress with this block (opening main page first then finish the page that is opened by deep link) and if we call `super.onBackPressed()` it will be infinite loop i think.